sklearn/utils/metaestimators.py

```diff
@@ -32,7 +52,11 @@ def __get__(self, obj, type=None):
         if obj is not None:
             # delegate only on instances, not the classes.
             # this is to allow access to the docstrings.
-            self.get_attribute(obj)
+            for item in self.attribute_name:
+                if _hasattr_nested(obj, item):
+                    break
+            else:
+                attrgetter(self.attribute_name[-1])(obj)
```

Comment:
@jnothman I modified the code to break and ignore other (later) delegates if the attribute has already been found. Does this implemented your suggestion?